product-service/src/main/java/com/extremecoder/productservice/controller/CategoryController.java

```diff
@@ -1,25 +1,39 @@
 package com.extremecoder.productservice.controller;
 
-import com.extremecoder.productservice.dto.CategoryDto;
+import com.extremecoder.productservice.dto.CategoryRegistrationRequest;
+import com.extremecoder.productservice.dto.Response;
 import com.extremecoder.productservice.service.CategoryService;
 import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.List;
+import javax.validation.Valid;
+import java.util.Date;
 
 @RestController
 @RequestMapping("/api/v1/categories")
 @RequiredArgsConstructor
-@Slf4j
```

Comment:
Please add @Slf4j here. We are already using lombok. And logging is also needed for the controller layer.